studymanager/src/main/resources/openapi/StudyManagerAPI.yaml

```diff
@@ -2052,6 +2180,27 @@ components:
         type: integer
         format: int32
       required: true
+    DateTime:
+      name: dateTime
+      in: query
+      schema:
+        type: string
+        format: date-time
+        readOnly: true
```

Comment:
`readOnly` parameters do not make sense:
```suggestion
```

Reply:
As discussed, I've deleted all readOnly arguments.